src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -135,6 +135,11 @@ function CustomFieldInputMultipleChoice(props) {
             {!props.readOnly && (
               <input
                 aria-labelledby={`${props.id}-label`}
+                aria-haspopup={'listbox'}
+                aria-expanded={renderPopup}
+                aria-autocomplete={'list'}
+                aria-controls={`${props.id}-multi-choice-listbox`}
```

Comment:
This makes sense to me now. 

I used to be confused as to why `aria-haspopup` would only indicate a role but never the actual element. This was the missing relationship attribute.